ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,72 @@
+Issue Title
+===========
+
+Your issue title is above. Please include the path to the
+topic with the issue. For example, if your issue is that
+our [list of samples](https://stage.docs.microsoft.com/en-us/dotnet/articles/samples-and-tutorials/index)
+is missing a sample, you'd put [samples-and-tutorials/index.md]
```

Comment:
should we be doing this instead? we're kind of asking them to know our structure. I prefer they'd add a link in the description and we can change the title later if we want.